extension/sumologicextension/extension.go

```diff
@@ -129,7 +130,16 @@ func newSumologicExtension(conf *Config, logger *zap.Logger, id component.ID, bu
 	var (
 		collectorName string
 		hashKey       = createHashKey(conf)
+		hashKeyV2     = createHashKeyV2(conf)
 	)
+
+	_, err = credentialsStore.Get(hashKeyV2)
+	if err != nil {
+		logger.Info("using v1 key as v2 credentials not found", zap.String("hashKeyV2", hashKeyV2), zap.Error(err))
+	} else {
+		logger.Info("v2 credentials found for key", zap.String("hashKeyV2", hashKeyV2))
```

Comment:
I think this log line should be at debug level since it is going to happen overwhelmingly often and be of little interest.